pkgs/development/libraries/tbox/libtbox.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@out@
```

Comment:
Please try to upstream it

Reply:
Upstream provides pkgconfig files via `xrepo`，use  `xmake install -o xxx` to get .pc file (and the binary）


We use [xmake.sh](https://github.com/xmake-io/xmake.sh) to build tbox, currently does not support exporting `.pc` file